### ISSUE TITLE
Regenerate composer autoload files

### ISSUE THREAD

**Issue by aszlig** [pulsejet/nextcloud-oidc-login#1]
Hi,

From looking at your code and comparing it against the [code of socialllogin](https://github.com/zorn-v/nextcloud-social-login) it looks very similar in structure, which might suggest that you used the app as a base.

Now since you're using the [same](https://github.com/pulsejet/nextcloud-oidc-login/blob/dee75a5f6582254c48f98f68877105dad20c7d86/3rdparty/composer/autoload_real.php#L22) generated composer autoload files as [`sociallogin`](https://github.com/zorn-v/nextcloud-social-login/blob/885963d0128f344a3b8a117e6bd6ec87e366bc0e/3rdparty/composer/autoload_real.php#L22), enabling both apps in Nextcloud will cause an error like this:

```
PHP Fatal error: Cannot declare class ComposerAutoloaderInit2fd16dd8fbb94748e1ada016e6921a0f, because the name is already in use in .../apps/sociallogin/3rdparty/composer/autoload_real.php on line 5
```

Cc: @zorn-v

**Comment by aszlig** [comment]
@pulsejet: Haha, I didn't actually read the [README](https://github.com/pulsejet/nextcloud-oidc-login/tree/dee75a5f6582254c48f98f68877105dad20c7d86#nextcloud-oidc-login) of your app, because it's even documented:

```
... Even though a fork of nextcloud-social-login, it fundamentally differs in two ways...
```

Still, this doesn't change anything about this issue: Please regenerate the composer autoload files, thanks :-)

**Comment by pulsejet** [comment]
@aszlig thanks for reporting this! Should be fixed with 3f8b4a92a01c9ce597515159d531861cb24245dd. Please reopen if I'm missing something.
